filesystem/File.php

```diff
@@ -390,7 +390,6 @@ public function getCMSFields() {
 
 		//get a tree listing with only folder, no files
 		$folderTree = new TreeDropdownField("ParentID", _t('AssetTableField.FOLDER','Folder'), 'Folder');
-		$folderTree->setChildrenMethod('ChildFolders');
```

Comment:
Oh, I see. I really prefer that treedropdownfield didn't have sepcial exemptions built in, but I guess we can't change that in 3.1.